src/seedu/addressbook/AddressBook.java

```diff
@@ -486,13 +486,24 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<HashMap<Per
      */
     private static ArrayList<HashMap<PersonProperty, String>> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<HashMap<PersonProperty, String>> matchedPersons = new ArrayList<>();
+        findPersons(keywords, matchedPersons);
+        return matchedPersons;
+    }
+
+    public static void findPersons(Collection<String> keywords,
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.